services/api/service.go

```diff
@@ -598,6 +605,7 @@ func (api *RelayAPI) processOptimisticBlock(opts blockSimOptions) {
 		"optBlocksInFlight": api.optimisticBlocksInFlight,
 	}).Infof("simulating optimistic block with hash: %v", opts.req.BuilderSubmitBlockRequest.BlockHash())
 	reqErr, simErr := api.simulateBlock(ctx, opts)
+	simResultC <- &blockSimResult{reqErr == nil, true, reqErr, simErr}
```

Comment:
i'd prefer if the fields would be explicitly set (i.e. use the property names), as this would make it clearer what each field means